texttospeech/synthesize_file/synthesize_file.go

```diff
@@ -0,0 +1,146 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package main
+
+/*
+Example usage:
+	go get -u cloud.google.com/go/texttospeech/apiv1
+	go run synthesize_file.go --text ../resources/hello.txt
+	go run synthesize_file.go --ssml ../resources/hello.ssml
+*/
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+
+	"golang.org/x/net/context"
+
+	texttospeech "cloud.google.com/go/texttospeech/apiv1"
+	texttospeechpb "google.golang.org/genproto/googleapis/cloud/texttospeech/v1"
+)
+
+// [START tts_synthesize_text_file]
+func SynthesizeTextFile(text_file, output_file string) error {
+	ctx := context.Background()
+
+	client, err := texttospeech.NewClient(ctx)
+	if err != nil {
+		return err
+	}
+
+	text, err := ioutil.ReadFile(text_file)
+	if err != nil {
+		return err
+	}
+
+	input := texttospeechpb.SynthesisInput{
+		InputSource: &texttospeechpb.SynthesisInput_Text{string(text)}}
+
+	// Note: the voice can also be specified by name.
+	// Names of voices can be retrieved with client.ListVoices().
+	voice := texttospeechpb.VoiceSelectionParams{
+		LanguageCode: "en-US",
+		SsmlGender: texttospeechpb.SsmlVoiceGender_FEMALE}
+
+	audioConfig := texttospeechpb.AudioConfig{
+		AudioEncoding: texttospeechpb.AudioEncoding_MP3}
+
+	request := texttospeechpb.SynthesizeSpeechRequest{
+		Input: &input,
+		Voice: &voice,
+		AudioConfig: &audioConfig,
+	}
+	response, err := client.SynthesizeSpeech(ctx, &request)
+	if err != nil {
+		return err
+	}
+
+	err = ioutil.WriteFile(output_file, response.AudioContent, 0644)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("Audio content written to file: %v\n", output_file)
+	return nil
+}
+// [END tts_synthesize_text_file]
+
+// [START tts_synthesize_ssml_file]
+func SynthesizeSsmlFile(ssml_file, output_file string) error {
```

Comment:
It can look a bit weird, but Go prefers to have make initialisms with consistent case. This would be `SynthesizeSSMLFile`. See https://github.com/golang/go/wiki/CodeReviewComments#initialisms.

Reply:
Got it, changed